config/locales/en/dashboard.yml

```diff
@@ -50,6 +51,7 @@ en:
       status:
         in_progress: In progress
         submitted: Submitted
+      no_records: There are no applications in progress
```

Comment:
This needs to be a different message, as these are submitted/returned applications. Not in progress.

Reply:
Just realised that I misinterpreted what was meant by "the solution is valid for all three statuses". I'll change that now.